pymeasure/adapters/prologix.py

```diff
@@ -37,8 +37,10 @@ class PrologixAdapter(VISAAdapter):
     Serial connection sharing is achieved by using the :meth:`.gpib`
     method to spawn new PrologixAdapters for different GPIB addresses.
 
-    :param port: The Serial port name or a connection object
-    :param address: Integer GPIB address of the desired instrument
+    :param resource_name: A
+        `VISA resource string <https://pyvisa.readthedocs.io/en/latest/introduction/names.html>`__
+        that identifies the target of the connection.
```

Comment:
```suggestion
        that identifies the serial port to interface the Prologix adapter. This is typically in the form of `ASRLn` with `n` as a positive integer number.
```

I suggest this change to make clear to users that this VISA address has nothing to do with the address of the instrument you would like to control.

Reply:
I took the documentation from the visa Adapter, but yes, it should be more distinguishable.

I don't want to fix it to the serial connection, as there are ethernet versions of the Prologix, if I interpret correctly the results of my research.

Reply:
There's an ethernet version, but this has never been tested/used with this driver. The original driver was based on `SerialAdapter` and the documentation we provide to the user never refers to the ethernet version. I agree that, in the future, it could be nice to add support to ethernet, after we test it with some real hardware.

Reply:
Please have a look at my modified texts.

Reply:
I see the changes and they looks good to me, but I am a bit hesitant to accept the changes that remove the references to the USB version of the Prologix adapter, until we have a proof that also other ethernet version is working.

Reply:
Should we add a line, that only the USB version is tested?

Reply:
This could be ok, but really this code was designed to work with usb version.
I don't have myself an ethernet version to test.
I see it communicates via port 1234 and apparently the command set is similar, if not identical, but here my preference (not strong) is to be conservative, as I understand this is the approach we normally follow, and avoid to expand the set of supported adapters type.

In any case, not a big deal, in my opinion, feel free to choose your preferred approach:
- mention that only usb is tested
- revert changes that remove explicit reference to usb version.

Thanks

Reply:
I'll add "USB-GPIB adapter", but leave the parameters as they are.